### PR TITLE
Allow strings as value for MessageEntity objects.

### LINKED ISSUE #25: Cannot add sign to a gauge value
It is actually possible to add a `+/-` to a gauge 

> Adding a sign to the gauge value will change the value, rather than setting it.
https://github.com/etsy/statsd/blob/master/docs/metric_types.md#gauges

But it impossible with the component because: 
- the `addToSend` method [casts $value as `int`
](https://github.com/M6Web/Statsd/blob/d705e9d53936d689df1893e449a25944524d1ade/src/M6Web/Component/Statsd/Client.php#L144)
- there is a check in `MessageEntity` to [verify that `$value` is an `init`](https://github.com/M6Web/Statsd/blob/d705e9d53936d689df1893e449a25944524d1ade/src/M6Web/Component/Statsd/MessageEntity.php#L73)

I can still use the `writeDatas` method directly but this is not satisfying. 

This should definitely be supported by the client, as this is a valid StatsD message. 
I think this can be implemented quite easily, without any BC breaks. 

I can submit a pull request.

### PR DESCRIPTION
**Pull request by mihai-amihailesei**
Closes #25

### CODE REVIEW

File: src/M6Web/Component/Statsd/MessageEntity.php [hunk 1]
```diff
@@ -60,7 +60,7 @@ protected function checkConstructor()
             throw new Exception('node and unit have to be a string');
         }
 
-        if (!is_int($this->value)) {
+        if (!is_int(intval($this->value))) {
             throw new Exception('value has to be an integer');
```
Comment (Oliboy50) [inline src/M6Web/Component/Statsd/MessageEntity.php:64]:
```suggestion
        if (!is_int($this->value) || !is_string($this->value)) {
            throw new Exception('value has to be an integer');
```

File: src/M6Web/Component/Statsd/Client.php [hunk 2]
```diff
@@ -158,7 +158,7 @@ public function getServerKey($stats)
     protected function addToSend($stats, $value, $sampleRate, $unit, $tags)
```
Comment (Oliboy50) [inline src/M6Web/Component/Statsd/Client.php:158]: please update the PHP doc above from `string` to `int|string` too 🙏

**Review by Oliboy50 — APPROVED** [review]

**Review by FloFrad — APPROVED** [review]

**Review by Benoit382 — APPROVED** [review]

**Comment by mihai-amihailesei** [comment]
Could you please merge and tag a new version?

**Comment by Oliboy50** [comment]
Thanks for your time, released in [v2.1.0](https://packagist.org/packages/m6web/statsd#v2.1.0)

**Pull request merged**